src/dsp_decaytime.jl

```diff
@@ -7,10 +7,10 @@ Get statistics on the logarhithmic of the tail of the `wvfs` in the interval (`s
 # Returns
 - `τ`: decay time in µs
```

Comment:
Can you add the new methods to the docsting?

Reply:
There is only one new method (function) that I introduced that takes tuples directly rather than the `DSPConfig` struct.

The original docstring for the function in the main branch of this repo does not correspond to the function itself, as it states `dsp_decay_times(wvfs::AbstractSamples, start::Real, stop::Real)`.

Same as with variable renaming, I don't know if I'm the person to be updating the docstring to this function, as I'm not an expert on what it does. I could do it, proposing my description here so that of course you and @theHenks could check if it's correct; but again, this would slow down this PR, while its point was to introduce a tiny modification to your code that makes the function more accessible outside the DSP routines that revolve around `DSPConfig`.

I can help you with renaming variables and updating docstrings, but I would prefer to do it in another PR.

Reply:
That would acutally be very nice if you could do that. I tried my best to update all docstrings, but I am sure I am missing some.

Reply:
And yes, a new PR would be nice.

Reply:
In that case I still propose to close this PR for just this one modification, submit an issue about docstrings, and then do a different PR on that; otherwise I would have to wait for a long time just to share important code about extracting LegendGeSim simulation parameters from data where I use this function.

Of course you can first test whether this modification breaks anything, just to be safe; although with this simple multiple dispatch it shouldn't, unless I made some stupid mistake ^^